CHANGELOG.md

```diff
@@ -42,7 +44,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 ### Fixed
 
 - Fixed the scripted fields disappear when the fields of the events index pattern was refreshed [#6237](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6237)
+  <<<<<<< HEAD
```

Comment:
There seem to be some unresolved conflicts in the changelog.

Reply:
Fixed here https://github.com/wazuh/wazuh-dashboard-plugins/pull/6558/commits/02387c47148deba990a6bdf2219a0a2c4f0d0d18.